container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/user_services_functions/start_user_services.go

```diff
@@ -753,6 +769,51 @@ func createStartServiceOperation(
 	}
 }
 
+// TODO - clean this up this is super janky, a way to handle compose  & volume
+func handleFilesToBeMovedForDockerCompose(ctx context.Context, dockerManager *docker_manager.DockerManager, containerImageName string, cmdArgs []string, entrypointArgs []string, filesToBeMoved map[string]string) ([]string, []string, error) {
```

Comment:
extremely nit pick but can we keep reference to `DockerCompose` in this module to components, ideally the code shouldn't reflect any knowledge of DockerCompose even though thats the purpose of the hack, especially if this  this hack remains in the codebase for some reason)

Reply:
`s/handleFilesToBeMovedForDockerCompose/handleFilesToBeMoved`